DHL/Client/Web.php

```diff
@@ -35,13 +35,13 @@ class Web
      * Url to services
      * @var string
      */
-    private $_stagingUrl = 'https://xmlpitest-ea.dhl.com/XMLShippingServlet?isUTF8Support=true';
+    private $_stagingUrl = 'https://xmlpitest-ea.dhl.com/XMLShippingServlet';
 
     /**
      * Url to services
      * @var string
      */
-    private $_productionUrl = 'https://xmlpi-ea.dhl.com/XMLShippingServlet?isUTF8Support=true';
```

Comment:
We're taking away UTF8 support here because we don't want it, but other users of this library might. What would be better is if we made UTF8 support an option. So, leave these two changes to the URLs, but in addition, update the constructor to allow the specifying of UTF8 support:

```php
public function __construct($mode = 'staging', $enableUtf8Support = true)
{
    if ($enableUtf8Support)
    {
        // ... append ?isUTF8Support=true to each of the URLs from above.
```

Here, we are making the default support be `true` so that it is backwards-compatible. Otherwise anyone who switched to ours would immediately lose UTF8 support.